app/plugins/ConductrPlugin.scala

```diff
@@ -0,0 +1,20 @@
+package plugins
+
+import com.typesafe.conductr.bundlelib.play.StatusService
+import com.typesafe.conductr.bundlelib.play.ConnectionContext.Implicits
+import play.api.{ Application, Plugin, Logger }
+import play.api.libs.concurrent.Execution
+
+class ConductrPlugin extends Plugin {
+  
+  override def onStart() {
+    import Implicits.defaultContext
+    StatusService.signalStartedOrExit()
+    Logger.info("Application has started")
```

Comment:
According to @pvlugter the logger is not yet injectable, at least the last time he looked. @jroper @richdougherty?

Meanwhile the above should be able to be changed to a module.